helm/etcd-kubernetes-resources-count-exporter/templates/policy-exceptions.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if .Values.kyvernoPolicyExceptions.enabled }}
+  {{- if .Capabilities.APIVersions.Has "kyverno.io/v2alpha1/PolicyException" -}}
+apiVersion: kyverno.io/v2alpha1
+kind: PolicyException
+metadata:
+  name: {{ tpl .Values.resource.default.name . }}-policy-exceptions
+  namespace: {{ .Values.kyvernoPolicyExceptions.namespace | default "kube-system" }}
+  labels:
+    {{- include "etcd-kubernetes-resources-count-exporter.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": "pre-install,pre-upgrade"
+spec:
+  exceptions:
+  - policyName: require-run-as-non-root-user
+    ruleNames:
+    - run-as-non-root-user
```

Comment:
Suggestion: according to [the dashboard](https://giantswarm.grafana.net/d/e5SZJRo4z/security-teams-overview?orgId=1&var-team=All&var-category=All&var-policy=All&var-app=etcd-kubernetes-resources-count-exporter&var-deployment=All&var-daemonset=All), this app is failing some other policies. Some of them should just be fixed in the chart itself, but others may also need exceptions. You might consider just getting that out of the way in this PR.

Reply:
I added more exceptions, I was able to verify we should have everything now. Thanks for pointing me to the dashboard. 

I tried to limit the exceptions as low as possible.